satpy/composites/__init__.py

```diff
@@ -647,6 +659,17 @@ def _get_reflectance(self, projectables, optional_datasets):
 class NIREmissivePartFromReflectance(NIRReflectance):
     """Get the emissive par of NIR bands."""
 
+    def __init__(self, **kwargs):
+        """Collect custom configuration values.
+
+        Args:
+            max_sza (float): Maximum solar zenith angle in degrees that is
+                considered valid and correctable. Default 95.0.
+
+        """
+        self.sunz_threshold = kwargs.get('sunz_threshold')
```

Comment:
```suggestion
        self.sunz_threshold = max_sza
```

Reply:
I didn't see this parameter `max_sza` actually. As it said default=95 I thought it must be something else. I have no idea what that might have been used for, but think it came from a cut and paste (error) from the sunz-corrector. I deleted it and kept sunz_threshold.